odoo/tools/translate.py

```diff
@@ -460,7 +460,7 @@ def quote(s):
                      .replace('\n', '\\n"\n"')
 
 re_escaped_char = re.compile(r"(\\.)")
-re_escaped_replacements = {'n': '\n', }
+re_escaped_replacements = {'n': '\n', 't': '\t',}
```

Comment:
Shouldn't we re-escape other like `\r`?

Reply:
Could indeed be an issue for people editing PO files on Windows (CRLF)

Reply:
If you use a windows program inserting `\r\n` in po files and read it on a linux server (or the opposite), I am expecting other errors to come up.
Before adding more use cases, would like to actually see the bug hitting us (not theoretically).

Reply:
OK then.